test/functional/lua/vim_spec.lua

```diff
@@ -569,7 +569,7 @@ describe('lua stdlib', function()
     vim.fn.setenv("A", 123)
     ]]
     eq('123', funcs.luaeval "vim.env.A")
-    eq(NIL, funcs.luaeval "vim.env.B")
+    eq(nil, funcs.luaeval "vim.env.B")
```

Comment:
both of these `luaeval` to `v:null`. You need to use `vim.env.B == nil` instead

Reply:
or check for `mpack.NIL`

```suggestion
    eq(mpack.NIL, funcs.luaeval "vim.env.B")
```

Reply:
That is just the existing code.

Reply:
What's werid is that my local tests are passing fine.